pkg/controller/servicemesh/memberroll/namespace_reconciler.go

```diff
@@ -121,7 +121,8 @@ func (r *namespaceReconciler) initializeNetworkingStrategy(ctx context.Context)
 						return fmt.Errorf("unsupported cluster network plugin: %s", networkPlugin)
 					}
 				} else {
-					log.Info("cluster network plugin not defined, skipping network configuration")
+					log.Info("cluster network plugin not defined, defaulting to NetworkPolicy", networkType)
+					r.networkingStrategy, err = newNetworkPolicyStrategy(ctx, r.Client, r.meshNamespace)
```

Comment:
if we return `err` after this the alert will go away

Reply:
fixed